TeslaMateAgile/appsettings.json

```diff
@@ -18,5 +18,10 @@
   "Awattar": {
     "BaseUrl": "https://api.awattar.de/v1",
     "VATMultiplier": 1.19
-  }
+  },
+  "Barry": {
+    "BaseUrl": "https://jsonrpc.barry.energy/json-rpc",
+    "BarryApiKey": "ApiKey",
+    "BarryMPID": "MeteringPointId  "
```

Comment:
If these are just placeholder values they can be removed here, the appsettings.json is for default values that the user doesn't generally need to touch e.g. the `BaseUrl` as you have added